libcnb-proc-macros/src/lib.rs

```diff
@@ -0,0 +1,67 @@
+use proc_macro::TokenStream;
+use quote::quote;
+use syn::parse::{Parse, ParseStream};
+use syn::parse_macro_input;
+use syn::Token;
+
```

Comment:
Could you enable the additional lints enabled in the other crates?
https://github.com/Malax/libcnb.rs/blob/a66f1f90f458867ccc1b59ff3b3cceeea80c1918/libcnb-data/src/lib.rs#L3-L7

(I'm looking forward to rust-lang/rust-clippy/issues/6625 et al)